GRANTS.yaml

```
@@ -57,6 +57,14 @@ by-covid:
     url: https://by-covid.org/
 
 
+dekcd:
+    name: German Competence Center Cloud Technologies for Data Management and Processing (de.KCD) 
+    short_name: "de.KCD"
+    joined: 2025-03
+    funding_id: "?
```

```suggestion
    funding_id: "16DKZ2072F"
```

I hope it the Förderkennzeichen.

---

@bgruening is there some URL that can be derived from this ID that is a page listing your grant on the funder's website/database?

for example for gallantries we had funding ID `2020-1-NL01-KA203-064717`, and can link to https://erasmus-plus.ec.europa.eu/projects/search/details/2020-1-NL01-KA203-064717 for the official grants page.

who is the funding body for this grant and is there a similar page for the grant? Then I can add support for that